src/main/java/kitchenpos/menu/domain/MenuProduct.java

```diff
@@ -0,0 +1,123 @@
+package kitchenpos.menu.domain;
+
+import java.util.Objects;
+import javax.persistence.Embedded;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.ForeignKey;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import kitchenpos.common.domain.BaseEntity;
+import kitchenpos.common.domain.Quantity;
+import kitchenpos.exception.InvalidArgumentException;
+import kitchenpos.common.domain.Price;
+import kitchenpos.product.domain.Product;
+
+@Entity
+public class MenuProduct extends BaseEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long seq;
+
+    @ManyToOne(fetch = FetchType.LAZY, optional = false)
+    @JoinColumn(name = "menu_id", foreignKey = @ForeignKey(name = "fk_menu_product_menu"))
+    private Menu menu;
+
+    @ManyToOne(fetch = FetchType.LAZY, optional = false)
+    @JoinColumn(name = "product_id", foreignKey = @ForeignKey(name = "fk_menu_product_product"))
+    private Product product;
+
+    @Embedded
+    private Quantity quantity;
+
+    protected MenuProduct() {
+    }
+
+    private MenuProduct(Product product, Long quantity) {
+        this.quantity = Quantity.valueOf(quantity);
+        setProduct(product);
+    }
+
+    public static MenuProduct of(Product product, Long quantity) {
+        return new MenuProduct(product, quantity);
+    }
+
+    public void relateMenu(Menu menu) {
+        validateMenu(menu);
+        if (this.menu != null) {
+            this.menu.removeMenuProduct(this);
+        }
+        this.menu = menu;
+        menu.addMenuProduct(this);
+    }
+
+    public boolean equalMenu(Menu menu) {
+        if (this.menu == null) {
+            return null == menu;
+        }
+        return this.menu.equals(menu);
+    }
+
+    public boolean equalMenuProduct(MenuProduct other) {
+        return equalMenu(other.menu) && product.equals(other.product) && quantity.equals(other.quantity);
+    }
+
+    public void removeMenu() {
+        this.menu = null;
+    }
+
+    public Price getPrice(){
```

Comment:
```suggestion
    public Price getPrice() {
```

IntelliJ Auto Formatting 기능을 사용하는 습관을 들여보아요.

The auto-formatting shortcut in IntelliJ

* For windows Ctrl + Alt + L.
* For ubuntu Ctrl + Alt + windows + L.
* For Mac Command + Option + L.
*